rosa_tutorials/rosa-mobb-configure-cluster-pull-images-aws-ecr-tutorial.adoc

```diff
@@ -0,0 +1,175 @@
+:_content-type: ASSEMBLY
+[id=“rosa-mobb-configure-cluster-pull-images-aws-ecr-tutorial”]
+= Tutorial: Configuring a ROSA cluster to pull images from AWS Elastic Container Registry
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-mobb-<topic>-tutorial
+
+toc::[]
+
+//Mobb content metadata
+//Brought into ROSA product docs 2023-09-20
+//---
+//date: '2022-04-26'
+//title: Configuring a ROSA cluster to pull images from AWS Elastic Container Registry (ECR)
+//weight: 1
+//tags: ["AWS", "ROSA"]
+//authors:
+//  - Kevin Collins
+//  - Byron Miller
+//---
+== Prerequisites
+
+* link:https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html[AWS CLI]
+* link:https://mirror.openshift.com/pub/openshift-v4/clients/ocp/[`oc` CLI] 4.11+
+* link:https://podman-desktop.io/[Podman Desktop]
+
+== Background
+
+Quick introduction by Ryan Niksch and Charlotte Fung on link:https://youtu.be/1PBFtpCIMBo[YouTube].
+
+There are two options to use to authenticate with AWS Elastic Container Registry (ECR) to pull images.
+
+The traditional method is to create a pull secret for ECR:
+
+.Example
+[source,terminal]
+----
+$ oc create secret docker-registry ecr-pull-secret  \
+  --docker-server=<registry id>.dkr.ecr.<region>.amazonaws.com  \
+  --docker-username=AWS --docker-password=$(aws ecr get-login-password) \
+  --namespace=hello-world
```

Comment:
There should be a step to create a namespace named `hello-world`, otherwise it will fail